packages/cursorless-engine/src/processTargets/modifiers/surroundingPair/index.ts

```diff
@@ -86,13 +86,16 @@ function processSurroundingPairCore(
     );
 
     if (containingScope != null) {
-      return findSurroundingPairTextBased(
+      const surroundingRange = findSurroundingPairTextBased(
         editor,
         range,
         containingScope[0].contentRange,
         delimiters,
         scopeType,
       );
+      if (surroundingRange != null) {
+        return surroundingRange;
+      }
```

Comment:
excellent catch 👍.  I would be tempted to do 

```
return findSurroundingPairTextBased(
        editor,
        range,
        null,
        delimiters,
        scopeType,
      );
```

directly when we fallback rather than falling through to rest of function as I'm not sure we want to hit that next try/catch block

Reply:
done